Cargo.toml

```diff
@@ -7,9 +7,15 @@ edition = "2021"
 path = "src/lib.rs"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+default = ["MysqlDB"]
+PostgresqlDB=[]
+MysqlDB=[]
+AnyDB=[]
```

Comment:
Feature names are generally snake case by convention.

Reply:
Will change this right away